downloads/id/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: cheat-sheet
+title: Buku Saku Git GitHub
+byline: Git adalah sistem manajemen revisi terdistribusi bersifat terbuka yang menyediakan aktivitas GitHub di laptop atau komputer pribadi Anda. Buku saku ini meringkas baris perintah instruksi-instruksi Git yang paling sering dipakai sebagai referensi singkat.
+leadingpath: ../
+---
+
+{% capture colOne %}
+## Pasang Git
+GitHub menyediakan klien bagi komputer, termasuk di dalamnya ada antarmuka grafis untuk tindakan repositori yang paling umum dilakukan dan tampilan konsol tempat menuliskan baris perintah untuk melaksanakan tindakan lebih lanjut.
+
+### GitHub untuk Windows
+http://windows.github.com
+
+### GitHub untuk Mac
+http://mac.github.com
+
+Distribusi Git untuk Linux dan sistem POSIX tersedia di situs resmi Git SCM.
+
+### Git untuk semua media
```

Comment:
"Git untuk semua sistem operasi" would be better, rather than using ambiguous words in your translation.